ajax/libs/gasparesganga-jquery-loading-overlay/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "gasparesganga-jquery-loading-overlay",
+  "version": "1.5.4",
+  "author": {
+    "name": "Gaspare Sganga",
+    "email": "contact@gasparesganga.com",
+    "url": "https://gasparesganga.com"
+  },
+  "description": "A flexible loading overlay jQuery Plugin",
+  "homepage": "https://gasparesganga.com/labs/jquery-loading-overlay/",
+  "keywords": [
+    "ecosystem:jquery",
+    "jquery-plugin",
+    "jquery",
+    "plugin",
+    "loading",
+    "overlay"
+  ],
+  "license": "MIT",
+  "filename": "loadingoverlay.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gasparesganga/jquery-loading-overlay.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gasparesganga/jquery-loading-overlay.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "loading*"
```

Comment:
Please turn to use `"loadingoverlay*"`. According to https://gasparesganga.com/labs/jquery-loading-overlay/#get-it, we don't need to host `.gif`. And remember to remove the file.